pkg/xds/envoy/listeners/v3/direct_response_configurer_test.go

```diff
@@ -0,0 +1,53 @@
+package v3_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	util_proto "github.com/kumahq/kuma/pkg/util/proto"
+	"github.com/kumahq/kuma/pkg/xds/envoy"
+	. "github.com/kumahq/kuma/pkg/xds/envoy/listeners"
+	v3 "github.com/kumahq/kuma/pkg/xds/envoy/listeners/v3"
+)
+
+var _ = Describe("DirectResponseConfigurer", func() {
+	It("", func() {
```

Comment:
is this missing a name?

```suggestion
	It("should correctly set up direct response", func() {
```

Reply:
changed